dissector/messages.md

```diff
@@ -1267,77 +1267,177 @@ Message have zero to two payloads.
     ```
 
   - **Notes:** The passwords are not sent in some models of cameras namely
-    RLC-410 4mp, RLC-410 5mp, RLC-520 (fw 200710) in these cases the passwords
-    are blank. In some older cameras that do not use encryption at all these
-    passwords are completely visible to any network sniffers. Even the "encrypted"
-    cameras only have weak encryption that is easily broken since the
-    decryption key is fixed and well-known.
+    RLC-410 4mp, RLC-410 5mp, RLC-520 (fw 200710), RLC-811A in these cases the
+    passwords are blank. In some older cameras that do not use encryption at
+    all these passwords are completely visible to any network sniffers. Even
+    the "encrypted" cameras only have weak encryption that is easily broken
+    since the decryption key is fixed and well-known.
 
-    - 67: `<ConfigFileInfo> (FW Upgrade)`
+- 59: `<UserList>`
 
-      - Client
+  - Client
+
+    - Header
+
+        FIXME
```

Comment:
I don't know how to find the header, perhaps someone could lend a hand?

The data in messages.md was taken directly from https://github.com/QuantumEntangledAndy/neolink/issues/342

Reply:
Just write something like `Standard header` I do it a few times in that file with the newer entries because it is always the same